html/pages/device.inc.php

```diff
@@ -229,6 +229,14 @@
                 </a>
               </li>');
 
+        if (@dbFetchCell("SELECT COUNT(stp_id) FROM stp WHERE device_id = '".$device['device_id']."'") > '0') {
```

Comment:
this should do not count all the id entries, because of  "WHERE device_id =" statement in the query.
As I know the COUNT function is called after WHERE, but you can correct me if I'm wrong (I'm not the SQL guru)